core/src/main/java/org/eclipse/hono/util/HonoDeviceNotifications.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 1.0 which is available at
+ * https://www.eclipse.org/legal/epl-v10.html
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ */
+
+package org.eclipse.hono.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * A helper for creating device notification events.
+ */
+public final class HonoDeviceNotifications {
```

Comment:
I do not understand the purpose of this class.
Given that you already created a class representing a notification, the factory method for creating it should probably be part of that class, shouldn't it?